cmd/talosctl/cmd/talos/dashboard/components/tables_test.go

```diff
@@ -0,0 +1,66 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package components_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/talos-systems/talos/cmd/talosctl/cmd/talos/dashboard/components"
+	"github.com/talos-systems/talos/cmd/talosctl/cmd/talos/dashboard/data"
+	"github.com/talos-systems/talos/pkg/machinery/api/common"
+	"github.com/talos-systems/talos/pkg/machinery/api/machine"
+)
+
+func TestUpdate(t *testing.T) {
```

Comment:
Will convert to table driven tests, just want to be sure that it makes sense  ...